Info boxes

Scott-Crawshaw opened the pull request:
Info boxes branch should be ready for a merge. Info boxes & their corresponding info are put into place, and a front page carousel with more details on the project is in place as well. Further steps might include coming up with more info box placements, and creating a tutorial walkthrough, so let's leave the branch open. I don't think this will break anything, but double check to make sure. I'll finalize the merge in class today

hack-heart commented on src/constants.js:
```diff
@@ -1,4 +1,4 @@
-// export const API_URL = 'http://localhost:3838/';
+// export const API_URL = 'https://med30.herokuapp.com/';
+export const API_URL = 'http://localhost:3838/';
 export const CALL = 'CALL';
```
Q: why this change?

Scott-Crawshaw replied:
Good call, I'll change that back

hack-heart commented on src/common/InfoText.js:
```diff
@@ -0,0 +1,84 @@
+// Front Page:
+export const frontPage = 'This application aims to discuss the inefficiencies in the storage and retrieval of medical record data. \
```
FIX: These strings shouldn't be variables. Most things on a website are strings, so it's much easier to place the strings where they should be. The only alternative is if there are multiple questions/answers all in the same format, then placing them in a JS object/array or a JSON file would make mapping them to a React component or fragment easier.

Scott-Crawshaw replied:
Sounds good, I'll move the strings into the code. I will leave this file though, as some of the strings have yet to be used, so we want to keep the content for later.

Scott-Crawshaw replied:
Actually we decided to keep it for now, we worked like this because Kevin was making/adjusting the text while I was coding the boxes, and since we will continue doing that, it's easier for now. Am happy to move these though down the road, just wanna get this merged in during our in class meeting

hack-heart commented on src/home/LandingPage.js:
```diff
@@ -3,8 +3,12 @@ import { Link } from 'react-router-dom';
 import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
+import Carousel from 'react-bootstrap/Carousel';
```
NIT: (Nits are generally my personal opinion/suggestions). Most users don't bother scrolling through carousels. Prefer to display info vertically, especially since the landing page is bare.

hack-heart commented on src/home/LandingPage.js:
```diff
@@ -3,8 +3,12 @@ import { Link } from 'react-router-dom';
 import Header from '../common/Header';
 import image from '../images/landing_img.jpg';
+import {
+  probPrompt, prob1, prob1Content, prob2, prob2Content, prob3, prob3Content, prob4, prob4Content, sol1Content, sol2Content, sol3Content, sol4Content,
```
This is how tedious importing can get when there are many variables.

hack-heart commented on src/index.js:
```diff
@@ -7,6 +7,7 @@ import App from './app';
 import store from './app/store';
 import theme from './theme';
 import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
NIT: Project already uses MUI, Bootstrap is a different look and feel.

hack-heart commented on src/patient/PatientDoctors.js:
```diff
@@ -114,6 +116,7 @@ const PatientDoctors = () => {
         </Table>
       </TableContainer>
+      <InfoPopover style={{ marginTop: '15px', marginBottom: '5px' }}>{addDoctorInfo}</InfoPopover>
       <Button onClick={handleOpen} disableElevation disableRipple disableFocusRipple style={btnStyle} sx={{ flexDirection: 'column' }}>
```
Q: Is this part of the tutorial?
FIX: Place above table

Scott-Crawshaw replied:
Moved it, thanks! Is what part of the tutorial? There wasn't much of a tutorial, so likely no

Scott-Crawshaw commented:
@hack-heart Thanks for the thorough comments, much appreciated! I moved the info box on the doctors page and reverted the constants file. Kevin and I have a workflow going with the variables file (he writes and edits the text, I place the boxes) so it's easiest to keep that file for now, but am happy to move the strings into the code down the road. Let me know if there are any issues
